### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [streamlink/streamlink#4096] [General]
Would a Mastodon account for this project be possible?

**Comment by gravyboat** [comment]
What purpose would this serve?

**Reply by linux-lukas** [reply]
I could think of three purposes:

* Primary: sharing project information
* Primary: Sharing project announcements
* Secondary: Interacting with the Fediverse community

**Reply by gravyboat** [reply]
That's not something we're interested in. Releases are announced on the release page, I'm not sure what project information would need to be shared, and we're already stretched thin. "Interacting" almost always turns in to support. We have gitter: https://gitter.im/streamlink/streamlink if people want to chat with each other.
